content/master/guides/import-existing-resources.md

```diff
@@ -5,7 +5,7 @@ weight: 200
 
 If you have resources that are already provisioned in a Provider,
 you can import them as managed resources and let Crossplane manage them.
-A managed resource's [`managementPolicies`]({{<ref "/v1.13/concepts/managed-resources#managementpolicies">}}) 
+A managed resource's [`managementPolicies`]({{<ref "/v1.16/concepts/managed-resources#managementpolicies">}})
```

Comment:
should this be set to `/latest/...` instead? do we have precedence for that with links in other places on this docs site?

Reply:
previously it was pinned to the version number so I went with that.

Reply:
fair enough - let's not rock the boat too much on this release then 😂